clrenv/lazy_env.py

```diff
@@ -196,10 +196,15 @@ def _apply_functions(d, recursive=False):
                     logger.warning(f"[{socket.gethostname()}] Offline, using placeholder value for {parameter_name}.")
                     value = OFFLINE_VALUE
                 else:
-                    value = _get_ssm_client().get_parameter(
-                        Name=parameter_name,
-                        WithDecryption=True
-                    )['Parameter']['Value']
+                    from botocore.exceptions import EndpointConnectionError
```

Comment:
nit: any reason not to move this import up to the top of the file? in theory, this doesn't seem like it should pull in anything else

Reply:
good point, thanks